docs/articles/Core/index.md

```diff
@@ -16,11 +16,11 @@ The PlatformToolkit is a stack of components and code shared over several platfo
 The dependency injection in MORYX is basically depending on a Castle Widsor Container and a MORYX specific decorator called CastelContainer.
 The CastelContainer will be used for the Component Composition as the DI Container for each level. Please open the following subpage to read more about the dependency injection in MORYX.
 
-- [DependencyInjection](xref:DependencyInjection)
+- [DependencyInjection](DependencyInjection)
 
 ## Configuration
 
-- [DelayQueue](xref:DelayQueue)
+- [DelayQueue](Collections/DelayQueue)
```

Comment:
same here

Reply:
you can check the links by just clicking on them in github. They should also work in Visual Studio code